docs/uds-packages/development/common-workflows.md

```diff
@@ -0,0 +1,43 @@
+# Common GitHub Workflows
+
+This repository is designed to provide common workflows for downstream UDS Packages to consume and provide inputs into to run their CI checks.
+This helps to ensure consistency and helps improve these repos easily over time as Renovate watches this repo for changes.
```

Comment:
Maybe change this to either "This ensure consistency and..." or "This helps to ensure consistency and enables easily improving there repos over time..."